data-service/src/test/java/com/amazonaws/blox/dataservice/integration/CreateEnvironmentIntegrationTest.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazonaws.blox.dataservice.integration;
+
+import static org.junit.Assert.assertEquals;
+
+import com.amazonaws.blox.dataservicemodel.v1.exception.ResourceExistsException;
+import com.amazonaws.blox.dataservicemodel.v1.model.EnvironmentId;
+import com.amazonaws.blox.dataservicemodel.v1.model.wrappers.CreateEnvironmentResponse;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class CreateEnvironmentIntegrationTest extends DataServiceIntegrationTestBase {
+  private static final String ACCOUNT_ID = "123456789012";
+  private static final String ENVIRONMENT_NAME = "environmentName";
+  private static final String CLUSTER_ONE = "cluster1";
+  private static final String CLUSTER_TWO = "cluster2";
+  private static final String TASK_DEFINITION = "taskDefinition";
+
+  DataServiceModelBuilder models = DataServiceModelBuilder.builder().build();
+  @Rule public ExpectedException thrown = ExpectedException.none();
+
+  @Test
+  public void testCreateEnvironmentSuccessful() throws Exception {
+    final EnvironmentId createdEnvironmentId =
+        models
+            .environmentId()
+            .accountId(ACCOUNT_ID)
+            .environmentName(ENVIRONMENT_NAME)
+            .cluster(CLUSTER_ONE)
+            .build();
+    // I create an environment
```

Comment:
This comment repeat below code,   I don't see the value to keep the cucumber sentences.